src/app/forms/templates/general/approval-type.template.ts

```diff
@@ -15,9 +15,9 @@ export function getTypeApprovalSection(vehicleType: VehicleTypes): FormNode {
         type: FormNodeTypes.CONTROL,
         editType: FormNodeEditTypes.SELECT,
         options: getOptionsFromEnum(approvalType),
-        validators: [
+        validators: (vehicleType === VehicleTypes.PSV ? [] : [
```

Comment:
Same as other PR, think this is mandatory for all vehicles

Reply:
Ok cool, that gets rid of one extended note...
Are they the same values?
Or does it need a different enum?